translations/inferno_devilish_plan.tex/cs.po

```diff
@@ -540,6 +659,12 @@ msgid ""
 "\\vspace*{\\fill}\n"
 "\n"
 msgstr ""
+"\\vspace*{\\fill}\n"
+"\\begin{center}\n"
+"\\includegraphics[width=0.3\\paperwidth, keepaspectratio]{\\art/slayer.png}\n"
+"\\end{center}\n"
+"\\vspace*{\\fill}\n"
+"\n"
```

Comment:
```suggestion
```
BTW, it's easier just to use `width=\linewidth` to fill up the width of the column. With those three other tweaks, everything fits nicely.